main_model.py

```diff
@@ -0,0 +1,101 @@
+def detect_document_type(document_path):
+    
+    guess_file = guess(document_path)
+    file_type = ""
+    image_types = ['jpg', 'jpeg', 'png', 'gif']
+    
+    if(guess_file.extension.lower() == "pdf"):
+        file_type = "pdf"
+        
+    elif(guess_file.extension.lower() in image_types):
+        file_type = "image"
+        
+    else:
+        file_type = "unkown"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the string "unknown". It should be corrected to ensure the function `detect_document_type` returns the correct value when the file type is not recognized.
```diff
-        file_type = "unkown"
+        file_type = "unknown"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->